mirgecom/multiphysics/__init__.py

```diff
@@ -26,7 +26,6 @@
 
 __doc__ = """
 .. automodule:: mirgecom.multiphysics.thermally_coupled_fluid_wall
-.. automodule:: mirgecom.multiphysics.phenolics
 
 .. autofunction:: make_interface_boundaries
 >>>>>>> main
```

Comment:
(Unrelated to your change, but while scrolling through I noticed the stray merge conflict annotation here. Could you delete it? 🙂)

Reply:
Done.